app/controllers/api/v1/accounts/contacts_controller.rb

```diff
@@ -30,10 +30,13 @@ def search
   end
 
   def import
+    render json: { error: 'File is blank' }, status: :unprocessable_entity and return if params[:import_file].blank?
```

Comment:
@murtaza-bagwala Would you be able to move this to the i18n file? `config/locales/en.yml`?

Reply:
Also the rubocop is failing, you can see the issues with `rubocop -a` command.

Reply:
sure @pranavrajs